### PR TITLE
Add sign up and login test

### PR DESCRIPTION
**Pull request by molinajulian**
## Summary

- Login test
- Sign up test

### CODE REVIEW

File: test/helpers/app.js [hunk 1]
```diff
@@ -10,8 +9,11 @@ const truncateTable = model =>
 
 exports.truncateDatabase = () => Promise.all(tables.map(truncateTable));
 
-exports.getResponse = ({ endpoint, header = {}, body, method = 'put' }) =>
-  request(app)
+exports.getResponse = ({ endpoint, header = {}, body, method = 'put' }) => {
+  // eslint-disable-next-line global-require
+  const app = require('../../app.js');
```
Comment (rodrigo103) [inline test/helpers/app.js]: Why this change?

Comment (molinajulian) [reply test/helpers/app.js]: Because all test are independent. Before this change, we used it the same app for all test, but I think that it's wrong. Each test must get response for any endpoint with your app.

File: test/helpers/faker.js [hunk 2]
```diff
@@ -1,12 +1,21 @@
 const faker = require('faker');
```
Comment (andreaceltich) [inline test/helpers/faker.js]: Maybe we can use factory.build on test for every model, we can customize it passing by parameter the attributes we want to replace to the generated. Also when using build, we are avoiding create it on the db and we can verify the object with schemas and avoid a new dependency :smiley:
